exercises/practice/microwave/microwave_test.rb

```diff
@@ -4,57 +4,39 @@
 # Common test data version: 1.7.0 cacf1f1
 class MicrowaveTest < Minitest::Test
   def test_one_second
-    # skip
     assert_equal '00:01', Microwave.new(1).timer
   end
 
   def test_fifty_nine_seconds
-    skip
     assert_equal '00:59', Microwave.new(59).timer
   end
 
   def test_sixty_seconds
-    skip
     assert_equal '01:00', Microwave.new(60).timer
   end
 
-  def test_one_minute
-    skip
-    assert_equal '01:00', Microwave.new(100).timer
-  end
-
   def test_ninety_seconds
-    skip
     assert_equal '01:30', Microwave.new(90).timer
   end
 
-  def test_one_minute_and_one_second
-    skip
-    assert_equal '01:01', Microwave.new(101).timer
-  end
-
   def test_sixty_one_seconds
-    skip
     assert_equal '01:01', Microwave.new(61).timer
   end
 
+  
   def test_one_minute_and_fifty_nine_seconds
-    skip
     assert_equal '01:59', Microwave.new(159).timer
   end
 
   def test_two_minutes
-    skip
-    assert_equal '02:00', Microwave.new(200).timer
+    assert_equal '02:00', Microwave.new(120).timer
```

Comment:
This is not true for the task at hand.  Notice 159 being 1 minute and 59 seconds, and 200 should be 2 minutes.  The only part that is an exception to this is the less than 100 entered keeps the number of minutes.

Reply:
So, now I got the idea. Thank you very much